packages/react-components/react-drawer/stories/src/Drawer/InlineDrawer.stories.tsx

```diff
@@ -58,29 +58,18 @@ type DrawerInlineExampleProps = DrawerProps & {
   setOpen: (open: boolean) => void;
 };
 
-const getMappedPosition = (position: DrawerProps['position']) => {
-  switch (position) {
-    case 'end':
-      return 'Right';
-
-    case 'bottom':
-      return 'Bottom';
-
-    default:
-      return 'Left';
-  }
-};
+const capitalize = (str?: string) => str && str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
nit for better type safety and clarity 

```suggestion
const capitalize = (str?: string): string => str ? str.charAt(0).toUpperCase() + str.slice(1) : ""; 
```